sdk/src/test/java/com/silanis/esl/sdk/internal/converter/DocumentPackageSettingsConverterTest.java

```diff
@@ -80,7 +74,7 @@ public void convertSDKToSDK() {
     public void convertAPIToAPI() {
 
         apiPackageSettings1 = createTypicalAPIPackageSettings();
-        apiPackageSettings2 = new DocumentPackageSettingsConverter(apiPackageSettings1).toAPIPackageSettings();
+        PackageSettings apiPackageSettings2 = new DocumentPackageSettingsConverter(apiPackageSettings1).toAPIPackageSettings();
```

Comment:
This section doesn't need to be modified.

Reply:
this variable is used only once within the method. they need not be class members.

Reply:
It is correct that it doesn't need to be a class variable.
But for some reasons, this correction is not necessary.
1. Your commits are completely irrelevant.
2. You only modified apiPackageSettings2. apiPackageSettings1 does not have to be a class variable either. I wonder why suddenly you just modified this.
3. This is not the only issue with It is not that only the DocumentPackageSettingsConverterTest has the issue. we need to modify all ConverterTest, if we decide to change.
As you can see, this is a unit test, not a product. It doesn't make much sense to spend time fixing unit tests. And this is not such an important issue at this time.

Reply:
apiPackageSettings1 is used multiple times in the class.